src/pyk/kcfg/kcfg.py

```diff
@@ -268,8 +274,8 @@ def uncovered(self) -> list[Node]:
         return [node for node in self.nodes if not self.is_covered(node.id)]
 
     @staticmethod
-    def from_claim(defn: KDefinition, claim: KClaim) -> tuple[KCFG, NodeIdLike, NodeIdLike]:
-        cfg = KCFG()
+    def from_claim(defn: KDefinition, claim: KClaim, cfg_dir: Path) -> tuple[KCFG, NodeIdLike, NodeIdLike]:
+        cfg = KCFG(cfg_dir=cfg_dir)
```

Comment:
Can we make `cfg_dir` optional here instead? Is there a reason its mandatory?

```suggestion
    def from_claim(defn: KDefinition, claim: KClaim, cfg_dir: Path | None = None) -> tuple[KCFG, NodeIdLike, NodeIdLike]:
        cfg = KCFG(cfg_dir=cfg_dir)
```

Reply:
Made it optional.